app/pulseapi/creators/models.py

```diff
@@ -28,7 +28,7 @@ class Creator(models.Model):
     """
     name = models.CharField(max_length=140)
     user = models.ForeignKey(
-        UserProfile,
+        settings.AUTH_USER_MODEL,
```

Comment:
I don't know what you mean with "users of the network site admin"; this makes Creators tied to the user model that is used to log people in irrespective of any particular app, it only looks at what Django + `settings.py` makes happen, there.